doctests/query-combined.js

```diff
@@ -0,0 +1,191 @@
+// EXAMPLE: query_combined
+// HIDE_START
+import assert from 'assert';
+import fs from 'fs';
+import { createClient } from 'redis';
+import { SchemaFieldTypes, VectorAlgorithms } from '@redis/search';
+import { pipeline } from '@xenova/transformers';
+
+const float32Buffer = (arr) => {
+  const floatArray = new Float32Array(arr);
+  const float32Buffer = Buffer.from(floatArray.buffer);
+  return float32Buffer;
+};
+
+async function embedText(sentence) {
+  let modelName = 'Xenova/all-MiniLM-L6-v2';
+  let pipe = await pipeline('feature-extraction', modelName);
+
+  let vectorOutput = await pipe(sentence, {
+      pooling: 'mean',
+      normalize: true,
+  });
+
+  const embedding = Object.values(vectorOutput?.data);
```

Comment:
I don't understand the change you want me to make here.

Reply:
I never used the @xenova/transformers package, but because you've used the `.?` I guess the function returns `undefined` in some cases (?)

Reply:
I dunno. I stole that code from another set of examples.

Reply:
I think @leibale point is that the pipeline can seemingly return an undefined and then you would be passing an undefined into Object.values(), which will result in an exception.

Now, if this were just to be a test, that probably wouldn't be a huge problem (as the exception would fail the test), but if this is also suppoesd to be documentation on how to use it, perhaps a check for it being undefined with a throw error if it is, would help (and then wouldn't need the ?, as the if undefined check would restrict it away).

just a thought, could be wrong :)

Reply:
I added a test around vectorOutput and removed the '?' from the assignment as suggested.